src/lib/viz/layer/Layer.ts

```diff
@@ -21,6 +22,16 @@ import { FunctionFilterApplicator } from '../filters/FunctionFilterApplicator';
 import { ColumnFilters } from '../filters/types';
 import { basicStyle } from '../style/helpers/basic-style';
 
+const DATA_READY_EVENT = 'layer:data:ready';
```

Comment:
deckgl uses 'camelCase', so how about 'layerDataReady'?

Reply:
Ok, I have not a real preference